app/src/main-process/menu/build-default-menu.ts

```diff
@@ -296,7 +297,13 @@ export function buildDefaultMenu(sharedProcess: SharedProcess): Electron.Menu {
   const showLogsItem: Electron.MenuItemOptions = {
     label: __DARWIN__ ? 'Show Logs in Finder' : 'S&how logs in Explorer',
     click() {
-      shell.showItemInFolder(getLogPath())
+      const logPath = getLogPath()
+      fileSystem.mkdir(logPath, function(err: NodeJS.ErrnoException): void {
+        if (err.code !== 'EEXIST') {
+          console.error(err)
```

Comment:
At the moment the two use cases behave slightly differently (one returns early, the other does not) but I'll leave it up to you to see if we can replace both usages cleanly.